docs/generic_webhook.md

```diff
@@ -0,0 +1,38 @@
+## Generic webhook request
+
+If the configuration option `generic_webhook_url` is set, then a synchronous request to
```

Comment:
synchronous?

Reply:
It started synchronous, but became async because we don't want to expose this to the end user.